src/stories/5-custom/iframe/iframe.stories.tsx

```diff
@@ -127,10 +128,10 @@ export const Overrides = OverrideStoryTemplate<IIframeSchema>("iframe-overrides"
 Overrides.args = {
 	referenceKey: "iframe",
 	validationTimeout: -1,
-	style: { border: "1px solid blue" },
+	style: { border: `${Border["width-010"]} ${Border.solid} blue` },
 	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--default-child`,
 	overrides: {
-		style: { border: "1px solid red" },
+		style: { border: `${Border["width-010"]} ${Border.solid} red` },
```

Comment:
you can revert the changes here, this is just an example for consumers and doesn't need to pull the tokens